### PR TITLE
Adding CucuShift tests to aws-ipi

### PR DESCRIPTION
**Pull request by jhou1**
This adds `@aws-ipi` test cases in the as `e2e-cucushift-aws-ipi` on Prow.

@liangxia @pruan-rht @dis016 @JianLi-RH PTAL.

### CODE REVIEW

File: ci-operator/step-registry/cucushift/aws/ipi/cucushift-aws-ipi-commands.sh [hunk 1]
```diff
@@ -0,0 +1,86 @@
...
+export USERS=${users}
+
+export E2E_SKIP_TAGS="not @flaky and not @inactive and not @upgrade-prepare and not @upgrade-check"
+
+cd verification-tests
+# run normal tests
+parallel_cucumber -n 4 --first-is-1 --type cucumber --prefix-output-with-test-env-number --exec \
+    'export OPENSHIFT_ENV_OCP4_USER_MANAGER_USERS=$(echo ${USERS} | cut -d "," -f ${TEST_ENV_NUMBER});
+     export WORKSPACE=/tmp/dir${TEST_ENV_NUMBER};
+     parallel_cucumber --group-by found --only-group ${TEST_ENV_NUMBER} -o "--tags \"@aws-ipi and not @admin and not @destructive and #{E2E_SKIP_TAGS}" -p junit"'
```
Comment (liangxia) [inline ci-operator/step-registry/cucushift/aws/ipi/cucushift-aws-ipi-commands.sh:81]: The `"` are unpaired.

Comment (jhou1) [reply ci-operator/step-registry/cucushift/aws/ipi/cucushift-aws-ipi-commands.sh]: Thanks, missed that after adding the `E2E_SKIP_TAGS` var, now updated.

File: ci-operator/step-registry/cucushift/pre/cucushift-pre-commands.sh [hunk 2]
```diff
@@ -0,0 +1,77 @@
...
+
+runtime_env=${SHARED_DIR}/runtime_env
+
+cat <<EOF >>${runtime_env}
+# cucumber setting
+export CUCUMBER_PUBLISH_QUIET=true
+
+# Export those parameters before running
+export OPENSHIFT_ENV_OCP4_HOSTS="${hosts}:lb"
+export OPENSHIFT_ENV_OCP4_ADMIN_CREDS_SPEC=${KUBECONFIG}
```
Comment (liangxia) [inline ci-operator/step-registry/cucushift/pre/cucushift-pre-commands.sh:71]:
```
     Given I switch to cluster admin pseudo user                                          # features/step_definitions/user.rb:13
      unknown credentials specification: /tmp/kubeconfig-356498767 (RuntimeError)
      /verification-tests/lib/admin_credentials.rb:96:in `get'
      /verification-tests/lib/environment.rb:113:in `admin'
      /verification-tests/lib/world.rb:124:in `admin'
      /verification-tests/features/step_definitions/user.rb:15:in `/^I switch to cluster admin pseudo user$/'
      features/routing/rate-limit.feature:8:in `I switch to cluster admin pseudo user' 
```
It should be `export OPENSHIFT_ENV_OCP4_ADMIN_CREDS_SPEC=file://${KUBECONFIG}`

Comment (jhou1) [reply ci-operator/step-registry/cucushift/pre/cucushift-pre-commands.sh]: Yes, thanks, I just fixed it this morning. Then I run into

```
    Scenario: Mirroring built image doesn't degrade scheme2 ,keep consistent SHA's

Given I save a htpasswd registry auth to the :combine_dockercfg clipboard

Message:

    kubeconfig does not exists (RuntimeError)
./lib/admin_credentials.rb:77:in `get'
./lib/admin_credentials.rb:94:in `get'
```

Comment (liangxia) [reply ci-operator/step-registry/cucushift/pre/cucushift-pre-commands.sh]: I think `${KUBECONFIG}` is `/tmp/kubeconfig-xxx`.
Currently using `export OPENSHIFT_ENV_OCP4_ADMIN_CREDS_SPEC=file:///${KUBECONFIG}`, so there is an extra `/` there.

Comment (jhou1) [reply ci-operator/step-registry/cucushift/pre/cucushift-pre-commands.sh]: The error `kubeconfig does not exists (RuntimeError)` was found when I added `export OPENSHIFT_ENV_OCP4_ADMIN_CREDS_SPEC=file://${KUBECONFIG}`.

I updated it to `///` following the comment in `lib/admin_credentials.rb`, let's see how that works...

Comment (jhou1) [reply ci-operator/step-registry/cucushift/pre/cucushift-pre-commands.sh]: Still fails, now made an update to copy ${KUBECONFIG} to `/tmp/kubeconfig` and load it from there.

**Pull request merged**